source/_header.erb

```diff
@@ -2,8 +2,7 @@
   <div id="site-title">
     <%= link_to image_tag('https://assets.lrug.org/images/el-rug-sidebar.png', alt: 'El Rug Logo', width: 65, height: 66), '/', class: 'header__branding' %>
 
-    <%= link_to 'LRUG', '/' %> :
-    london <span class="ruby"><%= link_to 'ruby', 'http://www.ruby-lang.org/en/' %></span> user group
+    <h1><%= link_to 'LRUG', '/' %> : London <span class="ruby"><%= link_to 'Ruby', 'http://www.ruby-lang.org/en/' %></span> User Group</h1>
```

Comment:
![image](https://user-images.githubusercontent.com/325384/93709944-9550d900-fb3a-11ea-8039-ce062bdc39f6.png)

The headings are in a bit of a funny order, but no `<h1>` isn't ideal. So I added one :)

![image](https://user-images.githubusercontent.com/325384/93710009-f678ac80-fb3a-11ea-847f-3dd5bac49e54.png)

I kind of liked the slightly bigger font size & heaver weight, what do you think?

Reply:
Yeah, it's nice.  No problem with making this an `<h1>` visually at least.  However, I think there would be a problem on pages other than the main index page as we'd end up with 2 `<h1>`s.  The main index page doesn't have a page title - we just go straight into the meetings index and render via `source/layouts/base.erb` which has no `<h1>`.  However, if we look at an individual meeting page, those go through `source/layouts/layout.erb` which _does_ have a page title `<h1>` so if we make the thing in the title an `<h1>` we'd have two on the page, and that's as bad as not having one, right?

But ...

As I look at this, and the markup around it, I wonder if the `<div id="header">` wrapper should just be be `<header>`?  That should let us have two `<h1>` elements on a page rendered via `source/layouts/layout.erb`. Which brings us back to the main index not having an `<h1>`.  I think we could make the `<div>` from `source/meetings/_meeting.erb` into an `<article>` and then make the existing `<h2>` for the meeting title into an `<h1>`.  Dunno what the semantics are of not having a "page-scoped" `<h1>` on a page with lots of `<article>`s though.  Maybe it's _as_ bad as having none, or having two.

Aside (pun intended): If we did any of this we should probably make `<div id="footer">` into a `<footer>` too, while we're at it.  Not sure how far we want to take this - the markup is pretty mid-late 2000s and `<div>`-heavy and we might disappear down a rabbit hole.

Reply:
I added `<header>`/`<footer>`/`<article>` tags around the various elements, and Lighthouse seemed pretty happy with that approach on both the homepage & subpages.